### ISSUE TITLE
Suggestion: UI Scaling

### ISSUE THREAD

**Issue by ryanbis** [binogure-studio/city-game-studio-i18n#435]
I'd like to see some way to scale the UI for larger resolutions. If I have a 2K monitor, the game shouldn't look like it's running at 480p. The UI I'm referring to are all the items around the HUD, like the buttons on the bottom, but most importantly, the info boxes on the left and right. I should be able to see more than 3 studios without having to scroll. Same with the games list. This is driving me nuts when playing, especially once you get a few studios going, and lots of games. If there's a way to change this currently, I apologize, I have not found it.

There's also this white bar at the top of the screen which seems to be pointless (it doesnt stretch all the way across). I decided to just show a screenshot. 

![image](https://user-images.githubusercontent.com/2172100/85138619-1386a200-b211-11ea-8455-cd7aa5d46bde.png)

**Comment by Redxpert** [comment]
yes, it happend the same to me, the ui is too big and idk what is the of that white bar
Also i think that up the right stats there should be another stadistic whith what theme is going up and what theme is going down

**Comment by xsellier** [comment]
I downscaled the UI lately, so if you can give the game another try, then update this ticket please.
Thank you.

**Comment by ryanbis** [comment]
The game is shaping up pretty well, good job. While I think the UI is better, I still would like to see the games list be smaller. I mean the cards inside that box on the left. Those appear to be the same size, and you still can't fit 3 of them without having to scroll the window. I would like to see somewhere around 5 or 6 fit in that window before having to scroll it. Same with the info box on the right hand side, graphs don't need to be so huge to get the point across (perhaps if its smaller, you can then click on the graph to bring up a larger display for that game?)

Also, this may be another issue but scrolling in that games list on the right gets annoying when you have ported your game to other platforms. I also have a tendency to push the window to the right because of edge scrolling (is there a way to turn off edge scrolling?).
